src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -190,31 +176,22 @@ export const Sidebar: React.FC<SidebarProps> = ({
     }
   }, [quoteCartId, removeCampaignCode])
 
-  const isNorwegianBundle = isBundle(offerData) && isNorwegian(offerData)
-  const discounts: Array<React.ReactNode> = [
-    ...(isNorwegianBundle ? [textKeys.SIDEBAR_NO_BUNDLE_CAMPAIGN_TEXT()] : []),
-    ...(campaignText ? [campaignText] : []),
-  ]
-
   const showRemoveCampaignButton = campaign != null
   const isDiscountPrice =
     campaign?.incentive?.__typename === 'MonthlyCostDeduction' ||
     campaign?.incentive?.__typename === 'PercentageDiscountMonths' ||
-    isNorwegianBundle
+    isNorwegianBundle(offerData)
```

Comment:
I'm open to suggestions for a variable name that would hold the value returned by `isNorwegianBundle` predicate, so I wouldn't need to calculate the value [twice](https://github.com/HedvigInsurance/web-onboarding/pull/727/files#diff-34ac2504068f1bf7af38914d4352369f38bad63521df594af26a944fb2425992R193).
It feels the correct name is the one we're already using for the function 🤣

Reply:
haha yeah I agree that on second though the name is more like a variable. Function could be called e.g. `checkIsNorwegianBundle`. But let's leave it for now :P